commit 89a002e41ee6e4ba590335d5eb2226d179daf20e
Author: Tom Hughes <tom@compton.nu>
Date:   Fri Jul 26 08:57:22 2019 +0100

    Fix tests for changes in Node.js 12.x

diff --git a/test/node_modules/hello_world/hello.cc b/test/node_modules/hello_world/hello.cc
index 89bfd7a..7065896 100644
--- a/test/node_modules/hello_world/hello.cc
+++ b/test/node_modules/hello_world/hello.cc
@@ -6,7 +6,7 @@ void Method(const Nan::FunctionCallbackInfo<v8::Value>& info) {
 
 void Init(v8::Local<v8::Object> exports) {
   exports->Set(Nan::New("hello").ToLocalChecked(),
-               Nan::New<v8::FunctionTemplate>(Method)->GetFunction());
+               Nan::GetFunction(Nan::New<v8::FunctionTemplate>(Method)).ToLocalChecked());
 }
 
 NODE_MODULE(hello, Init)
