node-gyp/commit/e6699d13cdcd6abe543b250f4297808912a4a188

diff -Nur a/test/node_modules/hello_world/hello.cc b/test/node_modules/hello_world/hello.cc
--- a/test/node_modules/hello_world/hello.cc	2017-03-16 04:25:31.000000000 +0800
+++ b/test/node_modules/hello_world/hello.cc	2022-03-16 19:53:46.935767424 +0800
@@ -1,12 +1,11 @@
 #include <nan.h>
 
-void Method(const Nan::FunctionCallbackInfo<v8::Value>& info) {
+NAN_METHOD(Method) {
   info.GetReturnValue().Set(Nan::New("world").ToLocalChecked());
 }
 
-void Init(v8::Local<v8::Object> exports) {
-  exports->Set(Nan::New("hello").ToLocalChecked(),
-               Nan::New<v8::FunctionTemplate>(Method)->GetFunction());
+NAN_MODULE_INIT(Init) {
+  Nan::SetMethod(target, "hello", Method);
 }
 
 NODE_MODULE(hello, Init)
